challenge/src/main/java/com/ifood/challenge/location/LocationController.java

```diff
@@ -5,20 +5,28 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.swing.plaf.synth.SynthMenuBarUI;
 import java.util.List;
 
 @RestController
-@RequestMapping(path = "feather/{number}")
+
+@RequestMapping(path = "feather/")
 public class LocationController {
     private final LocationService locationService;
 
     @Autowired
     public LocationController(LocationService locationService) {
         this.locationService = locationService;
     }
-    
+
     @GetMapping
+    public List homePage(){
+        return List.of("Hello World");
+    }
+    
+    @GetMapping(path="/{number}")
     public List<Location> getLocations(@PathVariable("number")int number){
-        return locationService.getLocation(number);
+        return List.of(locationService.getLocation(number));
```

Comment:
Why did you get rid of `locationService`? 🤔

Reply:
I just change the getLocation to return the array, before that it returned a "List.of".